src/main/java/gov/usgs/ngwmn/logic/WaterLevelStatistics.java

```diff
@@ -173,19 +144,67 @@ public JsonData calculate(Specifier spec, List<WLSample> samples) {
 		
 		return builder.build();
 	}
-	
 
-	protected void overallStats(List<WLSample> samples, List<WLSample> sortedByValue) {
-		overallStatistics.overallStats(samples, sortedByValue);
-		if (samples == null || samples.size() == 0) {
+
+	protected void overallLastestPercentile(List<WLSample> samplesByDate) {
```

Comment:
I think this should be `overallLatestPercentile` (small typo)

Reply:
fixed